src/CameraControls/useCenterGraph.ts

```diff
@@ -36,46 +78,45 @@ export const useCenterGraph = ({
   const nodes = useStore(state => state.nodes);
   const invalidate = useThree(state => state.invalidate);
   const { controls } = useCameraControls();
-
-  // Find the ideal spacing for focusing
-  const centerPadding = useMemo(() => {
-    const { maxX, maxY } = getLayoutCenter(nodes);
-    return Math.max(maxX, maxY);
-  }, [nodes]);
```

Comment:
I found the best behavior when always setting `fill = false` in `fitToBox()` below and keeping padding the consistent default `50`. So I removed this dynamic centerPadding

I didn't see any issues testing a bunch of graphs, but I don't have all the context for this so I could be missing something